src/components/ToolMenu/Draw/StylingDrawerButton/index.tsx

```diff
@@ -35,7 +41,7 @@ export const StylingButton: React.FC = (): JSX.Element => {
       onClick={showDrawer}
       icon={<FontAwesomeIcon icon={faPaintBrush} />}
     >
-      {t('StylingDrawer.pickColor')}
+      {!drawerVisibility?t('StylingDrawer.openColorPalette'):t('StylingDrawer.closeColorPalette')}
```

Comment:
```suggestion
      {!drawerVisibility ? t('StylingDrawer.openColorPalette') : t('StylingDrawer.closeColorPalette')}
```

Reply:
done in: [2c93f63](https://github.com/terrestris/shogun-gis-client/pull/1562/commits/2c93f63d742d1e0b3e3778b754180dccbc371a59)